### DISCUSSION TITLE
failFast setting is invalid

### DISCUSSION THREAD

**Discussion by gocpplua** [argoproj/argo-workflows#10311] [Q&A]
v3.4.2 version
When I use the[ dag-disable-failFast.yaml](https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/dag-disable-failFast.yaml
) configuration to create a workflow, after step C fails, BDE continues to execute instead of failing directly.

**Comment by agilgur5** [comment]
This was superseded by #10312
